src/mqt/bench/benchmarks/qiskit_application_finance/portfolioqaoa.py

```diff
@@ -1,3 +1,4 @@
+"""Portfolio QAOA benchmark definition."""
```

Comment:
There is a comment right below this top level one. Consider combining the two.

Reply:
Fixed it.